sdk/cosmos/azure-cosmos-table/pom.xml

```diff
@@ -0,0 +1,23 @@
+
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-cosmos-table</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0.0</version><!-- Need not change for every release-->
+  <modules>
+    <module>azure-cosmos-table</module>
```

Comment:
This modules section shouldn't be here, this should be added to the modules list [sdk/cosmos/pom.xml](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/pom.xml) with the rest of the modules.